From c3a446e55620b354e65d9ed93e2ea6b48330b4ee Mon Sep 17 00:00:00 2001
From: VPutys <vputys@witekio.com>
Date: Fri, 13 Oct 2023 10:24:52 +0100
Subject: [PATCH] removing jsonlib to work with kirkstone python sdk

---
 iotconnect-sdk-1.0/iotconnect/client/mqttclient.py | 2 --
 1 file changed, 2 deletions(-)

diff --git a/iotconnect-sdk-1.0/iotconnect/client/mqttclient.py b/iotconnect-sdk-1.0/iotconnect/client/mqttclient.py
index 8bf5d0b..917bd03 100644
--- a/iotconnect-sdk-1.0/iotconnect/client/mqttclient.py
+++ b/iotconnect-sdk-1.0/iotconnect/client/mqttclient.py
@@ -3,8 +3,6 @@ import os
 import ssl as ssl
 import paho.mqtt.client as mqtt
 import json
-if ('linux' in sys.platform) and (sys.version_info >=(3,5)):
-    import jsonlib
 import time
 from iotconnect.IoTConnectSDKException import IoTConnectSDKException
 
